app/models/checkin.rb

```diff
@@ -42,6 +42,10 @@ class Checkin < ApplicationRecord
     end
   end
 
+  def self.import(file, device)
+    system "rake checkins:import[#{file.path},#{device}] &"
```

Comment:
@dmitrytrager could you review this PR please, have linked the ticket it is related to, just want to make sure the rake task is an ok way to do this.